runner/dl_checkpoints.sh

```diff
@@ -79,7 +79,40 @@ function download_all_models() {
 
     # Download live-video-to-video models.
     huggingface-cli download KBlueLeaf/kohaku-v2.1 --include "*.safetensors" "*.json" "*.txt" --exclude ".onnx" ".onnx_data" --cache-dir models
-    huggingface-cli download KwaiVGI/LivePortrait --include "*.safetensors" "*.json" "*.txt" --exclude ".onnx" ".onnx_data" --cache-dir models
+    huggingface-cli download stabilityai/sd-turbo --include "*.safetensors" "*.json" "*.txt" --exclude ".onnx" ".onnx_data" --cache-dir models
+    huggingface-cli download warmshao/FasterLivePortrait --local-dir models/FasterLivePortrait--checkpoints
+}
+
+function build_tensorrt_models() {
+    download_all_models
+
+    printf "\nBuilding TensorRT models...\n"
+
+    BUILD_TRT='python app/live/StreamDiffusionWrapper/build_tensorrt.py'
+    docker run --rm -it -v ./models:/models --gpus all \
+        livepeer/ai-runner:live-app-streamdiffusion \
```

Comment:
```suggestion
        livepeer/ai-runner:live-app-streamdiffusion \
        -e PIPELINE=live-video-to-video -e MODEL_ID=streamdiffusion \
```
I had to include these two variables for the container to stay running while tensorrt builds. We probably need similar variables for LivePortrait too